src/Microsoft.ML.AutoML/Experiment/RecipeInference.cs

```diff
@@ -13,9 +13,9 @@ internal static class RecipeInference
         /// </summary>
         /// <returns>Array of viable learners.</returns>
         public static IEnumerable<SuggestedTrainer> AllowedTrainers(MLContext mlContext, TaskKind task,
-            ColumnInformation columnInfo, IEnumerable<TrainerName> trainerWhitelist)
+            ColumnInformation columnInfo, IEnumerable<TrainerName> trainerAllowList)
```

Comment:
@harishsk: This isn't a breaking change, right?

Reply:
If this is a public API, then it could be a source-breaking change for someone who used explicitly named arguments. It wouldn't be a binary-breaking change.

Reply:
AutoML isn't part of ML.NET's stable API. So this should be okay I think. Is that correct @sharwell?

---
In reply to: [461252764](https://github.com/dotnet/machinelearning/pull/5328#discussion_r461252764) [](ancestors = 461252764)

Reply:
I can't speak to what is or is not part of the stable public API. If this is not part of the stable public API, then it seems like it would be an easy change to approve.